userspace/libsinsp/events/sinsp_events.h

```diff
@@ -206,21 +206,38 @@ set<ppm_event_code> sinsp_state_event_set();
 
 /*!
   \brief Get the name of all the events provided in the set.
+  Note:
+
+  When setting resolve_generic to false exact event table names are returned for each
+  ppm_event w/ exception of "syscall" and "unknown" placeholder strings.
```

Comment:
i would remove the part of `unknown`, with `resolve` equal to false we will never resolve syscall names so we cannot get the `unknown` syscall name